package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -973,6 +973,16 @@ dnsmasq_start()
 	append_bool "$cfg" rapidcommit "--dhcp-rapid-commit"
 	append_bool "$cfg" scriptarp "--script-arp"
 
+	config_get add_mac "$cfg" add_mac
+	if [ "$add_mac" = "1" ]; then
+		xappend "--add-mac"
+	elif [ -n "$add_mac" ]; then
+		xappend "--add-mac=$add_mac"
+	fi
```

Comment:
How about..... this?

```sh
	config_get add_mac "$cfg" add_mac
	[ -n "$add_mac" ] && xappend "--add-mac${add_mac:+=$add_mac}"
```

Reply:
That pattern is already used here, https://github.com/openwrt/openwrt/blob/28ff4a9ebc9671cceb4544efc8d362dd3fef91c5/package/network/services/dnsmasq/files/dnsmasq.init#L1019-L1025

I'd rather preserve the style.